src/classes/BDI_SettingsUI_CTRL.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>38.0</apiVersion>
```

Comment:
I know the page needs to be 39 for slds but should the remaining new classes be 37?